golang/exercises/typeconversions/stringconverstions.go

```diff
@@ -1,23 +1,42 @@
 package typeconversions
 
-import "fmt"
+import (
+	"fmt"
+	"math"
+	"strconv"
+)
+
+func roundFloat(val float64, precision uint) float64 {
+	ratio := math.Pow(10, float64(precision))
+	return math.Round(val*ratio) / ratio
+}
 
 func stringToInt(str string) int {
 	// convert to int
+	res, err := strconv.Atoi(str)
+	if err != nil {
+		panic(err)
+	}
+	return res
 
-	return 0
 }
 
 func stringToFloat(str string) float64 {
 	// convert to float with 4 digits of precision
-
-	return 0
+	s, err := strconv.ParseFloat(str, 64)
+	if err != nil {
+		panic(err)
+	}
+	s = roundFloat(s, 4)
+	return s
 }
 
 func FloatToString(value float64) string {
 	// convert to float with 2 digits of precision
-
-	return "0"
+	value = roundFloat(value, 2)
```

Comment:
Nice job with the roundFloat function.

However, we can use `fmt.Sprintf` to handle the precision. We can use `"%.2f"` and `"%.4f"` verbs to set the precision.

So, it would be `fmt.Sprint("%.2f", value)`